Tensile/Common.py

```diff
@@ -737,7 +737,7 @@ def getArchitectureName(gfxName):
     #  - Tail loop can be unrolled up to InnerUnroll amount if AssertSummationElementMultiple%InnerUnroll==0
     #
     # 1 indicates no assertion (since all sizes are multiples of 1)
-    "AssertSummationElementMultiple": [1,2,4,8,16,32,64],
+    "AssertSummationElementMultiple": [1,2,4,8,16,32,64,128,256],
```

Comment:
These changes should be documented here. The name ASEM doesn't really imply anything to do with GSU so it's not intuitive behavior to have it dependent on anything other than the summation dimension. Makes sense to want to include the changes in ASEM rather than introduce another Assert, but it should definitely be documented in Common.py for future reference

Reply:
I have added some comment here.
Please check if it is OK or not.